utils/data_utils_test.py

```diff
@@ -4,11 +4,11 @@
 
 class TestLoadCord(unittest.TestCase):
 
-	def test_load_preprocessed_cord_from_dir(self):
-		cord_dir = "../data"
-		out = data_utils.load_preprocessed_cord_from_dir(cord_dir)
-		self.assertEqual(out, True)
+    def test_load_preprocessed_cord_from_dir(self):
+        cord_dir = "../data"
```

Comment:
best not to use relative directory paths at all in the code. you can use the constants module to build the file path in a more reusable way